src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -375,7 +375,7 @@ export default {
 			flssLoanSearch: {},
 			quickFiltersOptions: {
 				gender: [{
-					key: '',
+					key: 'all',
```

Comment:
Just out of curiosity, why the change to `all` instead of empty string?

Reply:
Looks like if we have an empty string it won't apply it as an `id` property for html elements. Since the filter pills are technically checkbox inputs (`<input>`) when we pass in `:id=option.key` and `option.key === ''` it doesn't apply an ID at all and seems to ignore empty string, so any interactions with "All genders" doesn't get considered as part of the checkbox list state

Reply:
Obviously could have worked around this but I thought it made sense to just change the `key` across the board instead of interpolating a random char/phrase or something similar

Reply:
Sounds good, thanks for the info, using `all` is fine.